[2024-04-08T10:35:09.670+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-08T10:29:54.815147+00:00 [queued]>
[2024-04-08T10:35:09.698+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-08T10:29:54.815147+00:00 [queued]>
[2024-04-08T10:35:09.698+0000] {taskinstance.py:2193} INFO - Starting attempt 3 of 3
[2024-04-08T10:35:09.728+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-08 10:29:54.815147+00:00
[2024-04-08T10:35:09.756+0000] {standard_task_runner.py:60} INFO - Started process 122274 to run task
[2024-04-08T10:35:09.768+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-08T10:29:54.815147+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpuwc9t6n1']
[2024-04-08T10:35:09.774+0000] {standard_task_runner.py:88} INFO - Job 16: Subtask get_files
[2024-04-08T10:35:09.841+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files manual__2024-04-08T10:29:54.815147+00:00 [running]> on host 8056db226622
[2024-04-08T10:35:10.004+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-08T10:29:54.815147+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-08T10:29:54.815147+00:00'
[2024-04-08T10:35:10.007+0000] {logging_mixin.py:188} INFO - 2 files found in /opt/***/dags/data
[2024-04-08T10:35:10.010+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_01.json']
[2024-04-08T10:35:10.060+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240408T102954, start_date=20240408T103509, end_date=20240408T103510
[2024-04-08T10:35:10.120+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-08T10:35:10.164+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
